gc/verbose/schema.xsd

```diff
@@ -810,6 +818,11 @@ xmlns:vgc="http://www.ibm.com/j9/verbosegc">
 		<attribute name="non-local-percent" type="integer" use="required" />
 	</complexType>
 
+	<complexType name="offheap">
+		<attribute name="objects" type="integer" use="required" />
+		<attribute name="allocatedBytes" type="integer" use="required" />
```

Comment:
i'd simplify, just 'bytes' (otherwise to be consistent with objects, should be allocatedObjects)